inspirehep/modules/disambiguation/search.py

```diff
@@ -349,7 +349,7 @@ def get_signature(uuid):
             "authors.orcid",
             "authors.profile",
             "authors.recid",
-            "authors.role",
+            "authors.contributor_roles[0].value",
```

Comment:
Does this work? Or when specifying `_source` you just put `"authors.contributor_roles.value"` ?